conode/conode.go

```diff
@@ -129,6 +149,48 @@ func setup(c *cli.Context) error {
 	if c.String("debug") != "" {
 		log.Fatal("[-] Debug option cannot be used for the 'setup' command")
 	}
+
+	if c.Bool("batch") {
```

Comment:
I'm not sure about the naming, it's not doing a batch task. Maybe `--non-interactive` (if `app.InteractiveConfig` is actually non-interactive).